packages/storage/src/indexed-db/index.ts

```diff
@@ -208,27 +209,40 @@ export class IndexedDb implements IndexedDbInterface {
     );
   }
 
-  async *iterateTransactionInfo() {
-    yield* new ReadableStream(
-      new IdbCursorSource(
-        this.db.transaction('TRANSACTION_INFO').store.openCursor(),
-        TransactionInfo,
-      ),
-    );
+  async *iterateTransactions() {
+    yield* new ReadableStream({
+      start: async cont => {
+        let cursor = await this.db.transaction('TRANSACTIONS').store.openCursor();
+        while (cursor) {
+          cont.enqueue(TransactionInfo.fromJson(cursor.value));
+          cursor = await cursor.continue();
+        }
+        cont.close();
+      },
+    });
```

Comment:
Any reason for this? If not, let's change back. If so, please add a code comment.

Reply:
Yeah, I'll change it back